gossipd/routing.c

```diff
@@ -2762,96 +2762,6 @@ struct route_hop **get_route(const tal_t *ctx, struct routing_state *rstate,
 	return hops;
 }
 
-void routing_failure(struct routing_state *rstate,
```

Comment:
Well that is a lot of code removed.  I suppose these are just holdovers of the old days when `pay` was far less sophisticated.